src/mscorlib/src/System/GC.cs

```diff
@@ -66,6 +66,13 @@ public static class GC
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
         internal static extern int SetGCLatencyMode(int newLatencyMode);
 
+        [MethodImplAttribute(MethodImplOptions.InternalCall)]
+        internal static extern void GetMemoryInfo(out uint highMemLoadThreshold,
+                                                  out ulong totalPhysicalMem,
+                                                  out uint lastRecordedMemLoad,
+                                                  out UIntPtr lastRecordedHeapSize,
+                                                  out UIntPtr lastRecordedFragmentation);
```

Comment:
Maybe leave a note (or a using-alias) that these UIntPtr are actually size_t?

Reply:
I'll do that.